playbooks/robusta_playbooks/event_enrichments.py

```diff
@@ -272,6 +272,34 @@ def pod_events_enricher(event: PodEvent, params: EventEnricherParams):
         )
 
 
+@action
+def enrich_pod_with_node_events(event: PodEvent, params: EventEnricherParams):
+    """
+    Given a Kubernetes pod, fetch related events in the near past for its node
+    """
+    pod = event.get_pod()
+    node = pod.get_node()
```

Comment:
Can we skip loading the node here? (it's just an expensive operation)
For the events, we only need the node name, which we already have
The node doesn't have a namespace, and the kind is static